### ISSUE TITLE
elmstatic init - syntax error

### ISSUE THREAD

**Issue by tgelu** [alexkorban/elmstatic#1]
I just ran
``npm install -g elmstatic``
and in a new directory
``elmstatic init``
as in the guide and I get the following
```
const slashesToDots = R.replace(new RegExp(Path.sep, "g"), ".")
                                ^
SyntaxError: Invalid regular expression: /\/: \ at end of pattern
    at RegExp (native)
    at Object.<anonymous> (...\AppData\Roaming\npm\node_modules\elmstatic\bin\index.js:25:33)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.runMain (module.js:604:10)
    at run (bootstrap_node.js:394:7)
    at startup (bootstrap_node.js:149:9)
    at bootstrap_node.js:509:3
```

**Comment by alexkorban** [comment]
Ah, that's the windows path separator causing problems. Will fix.

**Comment by alexkorban** [comment]
Fixed in 0.3.3. I tested on Windows and found no further issues. Please open another issue if something else is wrong.

**Comment by tgelu** [comment]
Thank you very much for the prompt fix! I had a hunch it was my windows machine :)
